examples/v4/timeseries-dataview.html

```diff
@@ -0,0 +1,158 @@
+      mainDiv.style.zIndex = 999;
+      var body = document.querySelector('body');
+      body.appendChild(mainDiv);
+
+      var width = mainDiv.clientWidth;
+      var divWidth = (width - (32 * 2 + (data.length - 1) * 2)) / data.length;
+      var max = Number.MIN_VALUE;
+
+      if (status === 'loading') {
+        var p = document.createElement('p');
+        p.innerText = 'LOADING...';
+        mainDiv.appendChild(p);
+      } else {
+        data.forEach(function (bin, index) {
+          var div = document.createElement('div');
+          div.style.position = 'absolute';
+          div.style.bottom = 0;
+          div.style.left = (index * divWidth + 32 + index * 2) + 'px';
+          div.style.height = (bin.normalized * 128) + 'px';
+          div.style.width = '' + divWidth + 'px';
+          div.style.backgroundColor = '#FF0000';
+          div.style.zIndex = 1100;
+          mainDiv.appendChild(div);
+        });
+      }
+    }
+
+    (async () => {
+      var i = 0; // Number of reload cycles.
+      var map = L.map('map').setView([40.4169, -3.7035], 11);
+      L.tileLayer('https://cartodb-basemaps-{s}.global.ssl.fastly.net/rastertiles/voyager_nolabels/{z}/{x}/{y}.png').addTo(map);
+
+      // Create the Client
+      var client = new carto.Client({
+        apiKey: '84fdbd587e4a942510270a48e843b4c1baa11e18',
+        username: 'cartojs-test'
+      });
+
+      client.on(carto.events.SUCCESS, function () {
+        console.log('Client reloaded', i++)
+      });
+
+      client.on(carto.events.ERROR, function () {
+        console.error('Client Error!', i++)
+      });
+
+      // Create a source
+      var source0 = new carto.source.SQL('SELECT * FROM airbnb_listings');
+      window.source0 = source0;
+
+      // Create a style
+      var style = new carto.style.CartoCSS(`
+        #layer {
+          marker-width: 10;
+          marker-fill: red;
+        }`
+      );
+
+      // Create a layer
+      var layer = new carto.layer.Layer(source0, style);
+
+      // Add the layers
+      client.addLayer(layer);
+
+      // Add the layer to the map
+      client.getLeafletLayer().addTo(map);
+
+      // Dataviews
+      var timeSeries = new carto.dataview.TimeSeries(source0, 'last_review', {
+        offset: 0,
+        aggregation: 'month'
+      });
+      window.timeSeries = timeSeries;
+
+      timeSeries.on('dataChanged', function (newData) {
+        console.log('Data changed', newData);
+        createHistogram({ data: timeSeries.getData().result });
+      });
+
+      timeSeries.on('statusChanged', function (newStatus, error) {
+        console.log('Status changed', newStatus, error);
+        if (newStatus === carto.dataview.status.LOADING) {
+          console.log('loading');
+        }
+        if (newStatus === carto.dataview.status.LOADED) {
+          console.log('loaded');
+        }
+        if (newStatus === carto.dataview.status.ERROR) {
+          console.log('error: ', error);
```

Comment:
Not sure why but example is throwing some errors:

![screen shot 2017-11-15 at 11 59 37](https://user-images.githubusercontent.com/390398/32832735-d18f1480-c9fc-11e7-8af0-27435ce21753.png)

Reply:
I think it has something to do with requests from the dataview that are being cancelled when panning the map.

Reply:
That shouldn't happen since `aborted` requests should not throw erros. I'll take a look.

Reply:
Got it. We're subscribed to `error` events in the internal model. Backbone always trigger an `error` event when an Ajax request fails.
In the issue `error handling 2` we've fixed this so remember to get back to test this example once it's merged.